controller/src/test/java/io/pravega/controller/store/stream/StreamMetadataStoreTest.java

```diff
@@ -476,6 +476,26 @@ public void updateTest() throws Exception {
     }
 
     @Test
```

Comment:
I see `join`s in this test case, better have a timeout, same for the test following this one.

Reply:
There is a global timeout in this test class.. Set to 10 seconds.